llvm/lib/Transforms/FuzzIntrospector/FuzzIntrospector.cpp

```diff
@@ -1132,3 +1184,234 @@ static RegisterStandardPasses
         PM.add(new FuzzIntrospector());
       });
 */
+
+void FuzzIntrospector::branchProfiler(Module &M) {
+
+  SmallPtrSet<const BasicBlock *, 32> visistedBBs;
+  SmallVector<const BasicBlock *, 32> worklist;
+  std::string outFileName = getNextLogFile() + ".branchProfile.yaml";
+  std::vector<BranchProfileEntry> outMap;
```

Comment:
I think in general we should stick to LLVM naming convention and have the first letter upper case

Reply:
The same applies to the other parts of the diff